settings/memcache.settings.php

```diff
@@ -85,5 +85,10 @@
 
     // Move locks to memcache.
     $settings['container_yamls'][] = DRUPAL_ROOT . '/../vendor/acquia/blt/settings/memcache.yml';
+
+    // Enable compression for PHP 7.
+    $settings['memcache']['options'] = [
+      Memcached::OPT_COMPRESSION => TRUE,
```

Comment:
According to the [PHP docs](http://php.net/manual/en/memcached.constants.php), the default value for this should already be true.

However, I can see that the Drupal Memcache module overrides this to FALSE, which probably isn't great for a multisite environment with shared hardware. So I agree that we should make it TRUE.

It looks like there's [already a push to enable compression](https://www.drupal.org/project/memcache/issues/2958403) with Drupal Memcache. I'd like add a comment here that links to that issue so that we know that this code can be removed once that patch is committed.

Reply:
Acquia is currently supporting that particular issue as part of their investment in a stable memcache release. (see https://www.drupal.org/project/memcache/issues/2989594). I have been more than involved with that effort so will coordinate any needed blt settings updates immediately before.